src/primitives/transaction.h

```diff
@@ -375,6 +383,8 @@ struct CMutableTransaction
     std::vector<CTxOut> vout;
     uint32_t nLockTime;
     std::vector<CPourTx> vpour;
+    uint256 joinSplitPubKey;
+    boost::array<unsigned char, 64> joinSplitSig;
```

Comment:
The only point for `joinsplit_sig_t` was to avoid a parsing error in the `READWRITE()` macro for serializing `CTransaction`. That being said I agree.